src/Player.h

```diff
@@ -0,0 +1,57 @@
+#ifndef EIGHT_MINUTE_EMPIRE_PLAYER_H
+#define EIGHT_MINUTE_EMPIRE_PLAYER_H
+
+#include <string>
+#include <vector>
+#include "BiddingFacility.h"
+#include "Map.h"
+
+// Forward Declaration
+class BiddingFacility;
+
+class Player {
+
+private:
+    // TODO: Add Player Hand of Game Cards - Requires Part 4 to be complete
+    int* coins;
+    int* armies;
+    int* cities;
+    int* dateOfBirth;
```

Comment:
Just a thought to make this easier. If the STL already has a function for date conversion to unixtime, the gamedriver could ask for full birth dates, and then pass them to `Player` as a unix timestamp so it would be super easy to convert. 

I'm not sure if any such function exists, though.

Reply:
For now I'm just going to ask for the year. We can revisit if the instructor wants us to be more accurate with this rule.